examples/sign.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const crypto = require('crypto');
+
+class Sample {
+  constructor() {};
+
+  sign(privateKey, message) {
+    const signature = crypto.sign('sha256', Buffer.from(message), {
+      key: privateKey,
+      padding: crypto.constants.RSA_PKCS1_PADDING,
+    });
+
+    return signature.toString('base64');
+  }
+
+  main() {
+    const privateKey = fs.readFileSync('test_private_key.pem', 'utf8', (err, data) => data);
```

Comment:
Looks like `err` isn't used?

Reply:
Yup, it's just the way the method works. Ideally we would do try catch error handling (or even read streams), but since this is just an example I wanted to keep it as concise as possible.